tests/coupled/multibody/test_double_pendulum_slanted.py

```diff
@@ -0,0 +1,337 @@
+import numpy as np
+import unittest
+import os
+import shutil                 
+from sharpy.utils.constants import deg2rad
+
+class TestDoublePendulumSlanted(unittest.TestCase):
```

Comment:
I know this is how we do unit tests in aerospace. But, this is not a unit test by the normal definition, as it should use [mocking](https://docs.python.org/3/library/unittest.mock.html) for the other components. This is technically an integration test. I'm ngl, I still haven't figured out what's an effective method for testing scientific code is 🤷‍♀️

Reply:
This is how other tests are defined, so even if there is a better way to do this, I would rather they be consistent for now. We can look into swapping them all to a better definition in a future release.

Reply:
indeed @wong-hl - I came across this when I was looking for a way to have a parametrised `setup()` within uniitest - haven't heard about mocking so I'll read up on it!